### PR TITLE
chore: Refactored client to React

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR includes the refactoring of the client and partials to React.
- All Nunjucks partials are reworked to components
- client are reworked to React

### CODE REVIEW

File: __react/template/src/index.ts.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      import * as ${pascalMessageName}Message from "./messages/${pascalMessageName}";
+      export {${pascalMessageName}Message};
+    `);
+  }
+
+  return (
+    <File name="index.ts">
+      {
+        `
+        import {fromSeed} from 'ts-nkeys';
```
Comment (magicmatatjahu) [inline __react/template/src/index.ts.js:91]: I know about issues with indentation, but remember, everything in this file will be intend :)

Comment (magicmatatjahu) [reply __react/template/src/index.ts.js:91]: Also split imports and exports to seperate groups in file:

```ts
import ...
import ...

export ...
export...
```

**Review by magicmatatjahu — APPROVED** [review]
Only 2 suggestions :)

File: __react/template/src/index.ts.js (hunk 1 above)
Comment (jonaslagoni) [reply __react/template/src/index.ts.js]: I like the idea 😄

File: __react/template/src/index.ts.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+export declare interface NatsAsyncApiClient {
+  ${Events()}
+}
+
+export class NatsAsyncApiClient extends events.EventEmitter{
+  ${Standard(asyncapi)}
+  ${getChannelWrappers(asyncapi, params).join('')}
+}
```
Comment (magicmatatjahu) [inline __react/template/src/index.ts.js:139]: I meaned imports and exports groups in this whole block :)

Comment (jonaslagoni) [reply __react/template/src/index.ts.js:139]: ahh gotcha, changing.

**Review by magicmatatjahu — APPROVED** [review]

**Comment by jonaslagoni** [comment]
🙇

**Pull request merged**